cluster/ci/config/prow/plugins.yaml

```diff
@@ -277,6 +277,7 @@ plugins:
   - trigger
 
   openshift/installer:
+  - docs-no-retest
```

Comment:
Doesn't add any value now that we have tide

Reply:
Why is that? (Sorry, I'm new to these tools)